screens/Collection/Collection.tsx

```diff
@@ -1,9 +1,109 @@
-import React from 'react';
+import React, { useState } from 'react';
 
-import {CollectionContainer} from './Collection.style'
+import {
+  Title,
+  SubContainer,
+  NewCollectionButton,
+  NewCollectionButtonText,
+  SubContainerText,
+  GridWrapper
+} from './Collection.style';
+import PlusIcon from '../../assets/icons/plus_icon.svg';
+import { CollectionCard, Grid, ContainerWithSafeArea} from '../../components';
 
-interface CollectionProps {}
+interface CollectionProps {
+  navigation: any;
+}
+
+const testData = [
+  {
+    id: 1,
+    title: 'Hello',
+    review: 12,
+  },
+  {
+    id: 2,
+    title: 'Hello',
+    review: 12,
+  },
+  {
+    id: 3,
+    title: 'Hello',
+    review: 12,
+  },
+  {
+    id: 4,
+    title: 'Hello',
+    review: 12,
+  },
+  {
+    id: 5,
+    title: 'Hello',
+    review: 12,
+  },
+  {
+    id: 6,
+    title: 'Hello',
+    review: 12,
+  },
+  {
+    id: 7,
+    title: 'Hello',
+    review: 12,
+  },
+  {
+    id: 8,
+    title: 'Hello',
+    review: 12,
+  },
+  {
+    id: 9,
+    title: 'Hello',
+    review: 12,
+  },
+];
 
 export const Collection = (props: CollectionProps) => {
-  return <CollectionContainer>Collection</CollectionContainer>;
-};
\ No newline at end of file
+  const { navigation } = props;
+  const [modalVisible, setModalVisible] = useState(false);
+
+  const onDeleteHandle = () => {};
+
+  return (
+    <ContainerWithSafeArea>
+      <Title>คอลเลกชั่น</Title>
```

Comment:
`คอลเลกชัน` ไม่มีไม้เอก ฝากแก้คำตรงอื่นด้วยนะ

```suggestion
      <Title>คอลเลกชัน</Title>
```